src/_playground/Routes.js

```diff
@@ -345,7 +370,12 @@ export class Routes extends Component {
                             {navItems}
                         </nav>
                     </div>
-                    <div className='frDocs-Content'>
+                    <div className={docsContentClasses}>
+                        <button
```

Comment:
Should we be using the fundamental-react `Button` component here rather than a plain `<button>`.  I think you'd get a lot of the className assistance for free (or at least with props).